Core/Contributions/Solutions Software/EditableListView.cls

```diff
@@ -475,28 +469,6 @@ itemAndColumnAt: aPoint
 		ifTrue: [itemIndex@columnIndex]
 		ifFalse: [nil]!
 
-itemIndex: anInteger hasFocus: aBoolean
```

Comment:
What was the motivation for removing these completely? I'm not familiar with this control, but the method categorisation suggests they were considered part of the public interface.